pkg/addons/data/ConfigMap/openshift-metrics/prometheus.yaml

```diff
@@ -35,6 +35,8 @@ data:
       scheme: https
       tls_config:
         ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        # TODO: Fix me
```

Comment:
Copy-paste from previous commit comment

@jim-minter this does not work due certificate. By default, endpoints point to local address. In our case EP points to IP https://10.244.x.xxx:443/metrics

And certificate is valid only for:
Get https://10.244.x.xxx:443/metrics: x509: certificate is valid for 172.30.0.1, not 10.244.x.xxx

kubernetes 10.244.x.xxx:8053,10.244.x.xxx:8053,10.244.x.xxx:443 28m

Reply:
I've been here before with the template service broker (see a long way down in this file).  Does setting tls_config: server_name: kubernetes help here instead of the insecure_skip_verify?